hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/dynamicconfig/AddScheduledExecutorConfigMessageTask.java

```diff
@@ -58,6 +58,9 @@ protected IdentifiedDataSerializable getConfig() {
         config.setStatisticsEnabled(statsEnabled);
         MergePolicyConfig mergePolicyConfig = mergePolicyConfig(parameters.mergePolicy, parameters.mergeBatchSize);
         config.setMergePolicyConfig(mergePolicyConfig);
+        if (parameters.isCapacityPolicyExists) {
+            config.setCapacityPolicy(ScheduledExecutorConfig.CapacityPolicy.getById(parameters.capacityPolicy));
```

Comment:
Can you add a test for this enum to `EnumCompatibilityTest`?

Reply:
Added it. Thanks for detecting its lack.